README.md

```diff
@@ -10,6 +10,10 @@ Run 'grunt' to make sure gulp is installed and available.
 
 Run 'grunt devmode' to watch files, process scss to css and perform other tasks.
 
+### Adding new files
+If you add a partial "\_newfile.scss" 
+inside scss/components folder, it will be included automatically.
```

Comment:
don't you also need to add it to components.scss?

Reply:
i updated this